docs/topics/api/accounts.rst

```diff
@@ -72,7 +72,65 @@ This endpoint is a shortcut to your own account. It returns an :ref:`account obj
 
 .. http:get:: /api/v3/accounts/profile/
 
-    .. _self-account-object:
+
+----------------
+Collections List
```

Comment:
Following our typical API nomenclature, I'd prefer:

Find collections from a given user (If you have the right permission)

```
GET /api/v3/collections/collection/?user=123
```

Get a specific collection:

```
GET /api/v3/collections/collection/321/
```

I don't think it's a good idea to make `collections` a child of `accounts` - I made that mistake initially when doing the first iteration of the `reviews` API and it turned out to be a bad idea, it's too rigid, we might want other kind of filtering.

I also don't think it's worth doing a specific API for your own collections - I don't like having duplicate endpoints for the same resource. In that particular case we could do a redirect (ala `/accounts/profile/`) if it proves to be really useful.